src/helm/benchmark/vlm_run_specs.py

```diff
@@ -3,6 +3,7 @@
 from .adaptation.adapter_spec import AdapterSpec
 from .adaptation.adapters.adapter_factory import ADAPT_GENERATION_MULTIMODAL, ADAPT_MULTIPLE_CHOICE_JOINT_MULTIMODAL
 from .metrics.metric import MetricSpec
+from .metrics.vision_language.image_metrics import ImageMetric
```

Comment:
This import needs to be moved into `get_image2latex_latex_spec()`

In general, importing a run_specs model should not result in an import of a metrics or scenario module.

Reply:
Done